devices/Bmm150/Bmm150I2c.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Magnetometer
+{
+    /// <summary>
+    /// Default I2C interface for the Bmm150
+    /// </summary>
+    public class Bmm150I2c : Bmm150I2cBase
```

Comment:
why is this class needed?

Reply:
We keep this implementation (same as AK8963).